### PR TITLE
Add Hotpod Yoga story

### PR DESCRIPTION
**Pull request by Ken-Lo**
screen shot 2016-07-18 at 15 55 44

### CODE REVIEW

**Comment by Ken-Lo** [comment]
Hey @swillis - could we get your thoughts on this? Will start adding additional stories soon.

File: app/pages/stories/story-page-new.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default class StoryPageNew extends React.Component {
   static propTypes = {
     title: React.PropTypes.string.isRequired,
     benefits: React.PropTypes.string.isRequired,
+    position: React.PropTypes.string,
```
Comment (Ken-Lo) [inline app/pages/stories/story-page-new.js]: @swillis Just to bring your attention to this: I added a new attribute property called `position` (e.g. `Co-Founder`, `CFO`, `Payments Team Lead`, etc).

Comment (swillis) [inline app/pages/stories/story-page-new.js]: Looks good! Although could we just make it required so we don't need this little function? Just an idea, happy to leave it with you 😄

**Comment by swillis** [comment]
@Ken-Lo: Looks good! You've got a commit to squash, but otherwise ready to go 👍 

Let me know if you want help doing the squash 😄

File: app/pages/stories/story-page-new.js (hunk 1 above)
Comment (Ken-Lo) [inline app/pages/stories/story-page-new.js]: Thanks Sam! I will leave this as optional for the time being - will review once again when we've added in the other case studies as not sure we have a position for them all :)

**Comment by Ken-Lo** [comment]
Thanks Sam - much appreciated!

**Pull request merged**